app/questionnaire/questionnaire_schema.py

```diff
@@ -62,7 +62,7 @@ def __init__(
     ):
         self._parent_id_map: dict[str, str] = {}
         self._list_name_to_section_map: dict[str, list[str]] = {}
-        self._list_name_to_section_id_origin_map: dict[str, str] = {}
+        self._list_name_to_section_id_origin_map: dict[str, list] = {}
```

Comment:
I suggest updating this to something like `self._list_collector_section_ids_by_list_name`. Open to other suggestions.

Reply:
Updated 👍